nvidia_triton/tests/test_unit.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.base.constants import ServiceCheck
+from datadog_checks.nvidia_triton import NvidiaTritonCheck
+
+from .common import METRICS_MOCK, get_fixture_path
+
+
+def test_check_metrics_nvidia_triton(dd_run_check, aggregator, instance_metrics, mock_http_response):
+    """
+    The instance is a deepcopy of the INSTANCE_DISABLED_SERVER_INFO in common
```

Comment:
Please either delete this or replace it with sentence that describes the test in plain english, the way you do for the service check tests